Source/Tx.Network/Syslogs/SyslogParser.cs

```diff
@@ -1,47 +1,22 @@
 namespace Tx.Network.Syslogs
 {
     using System;
-    using System.Collections.Generic;
     using System.Text;
     using System.Text.RegularExpressions;
     using Tx.Network;
 
-    public class SyslogParser
+    public static class SyslogParser
     {
         public static readonly Regex DefaultParser = new Regex(
             @"\<(?<PRIVAL>\d+?)\>\s*(?<MESSAGE>.+)",
             RegexOptions.CultureInvariant | RegexOptions.ExplicitCapture | RegexOptions.Compiled);
 
-        private readonly Regex parser;
-
-        private readonly string[] groupNames;
-
-        private readonly bool usingOnlyDefaultParser;
-
-        public SyslogParser()
-            : this(DefaultParser)
+        public static Syslog Parse(IUdpDatagram receivedPacket)
         {
-            this.usingOnlyDefaultParser = true;
+            return Parse(receivedPacket.Data.AsByteArraySegment(), receivedPacket.ReceivedTime, receivedPacket.PacketHeader.SourceIpAddress.ToString());
         }
 
-        public SyslogParser(Regex parser)
-        {
-            if (parser == null)
-            {
-                throw new ArgumentNullException("parser");
-            }
-
-            this.parser = parser;
```

Comment:
I would use the Rx and Tx naming convention with _parser as opposed to this.parser
This is as easy (or easier) to get intellisensse and also readable